### PR TITLE
feat(contact): add contact page

### PR DESCRIPTION
**Pull request by ncoden**
Add single contact page.

**Other changes**:
- Add tag (`.co-tag`) and list (`.co-text-list`) components
- Make `.co-avatar` modifiers extendable
- Add required en/fr translations

**Previews**:
PC preview
Mobile preview

### CODE REVIEW

File: src/js/directive/contact.js [hunk 1]
```diff
@@ -33,7 +31,93 @@ export function ContactDirective() {
...
+                <widget-contact-avatar-letter
+                  contact="ctrl.contact"
+                ></widget-contact-avatar-letter>
+              </div>
+
+              <h2 class="co-contact__datas-irl__name">
+                {{ ctrl.contact.title }}
+                <span class="badge">
+                  <span class="fa fa-lock"></span>
+                  {{ }}
```
Comment (iamdey) [inline src/js/directive/contact.js]: o/

Comment (ncoden) [inline src/js/directive/contact.js]: I know, but there is no "privacy index" given by the API. So this tag is empty, to show I wait the number here, but don't know its (future) name.

Comment (ncoden) [inline src/js/directive/contact.js]: It's wonderfully dirty, i know

Comment (iamdey) [inline src/js/directive/contact.js]: I prefer an old fashion html comment or a fake PI. 
I don't know if angular do weird stuff when compiling this "no value"

Comment (ncoden) [inline src/js/directive/contact.js]: Angular ignore it. But i'll replace it with a html comment

**Pull request merged**